modules/lib/src/main/resources/assets/js/app/inputtype/ui/selector/image/ImageEditor.ts

```diff
@@ -2285,4 +2315,17 @@ export class ImageEditor
     private notifyImageError(event: UIEvent) {
         this.imageErrorListeners.forEach(listener => listener(event));
     }
+
+    private getEditResetLabel(): string {
+        const item = new ResponsiveItem(this);
+
+        let resetLabel = i18n('button.reset');
+
+        if(item.isInRangeOrBigger(ResponsiveRanges._360_540)){
+            if(this.isFocusEditMode()) { resetLabel = i18n('editor.resetautofocus'); }
+            if(this.isCropEditMode()) { resetLabel = i18n('editor.resetmask'); }
```

Comment:
Don't inline condition body, it should be on a separate line even if it's one line.

```Typescript
if (item.isInRangeOrBigger(ResponsiveRanges._360_540)){
  if (this.isFocusEditMode()) {
    return i18n('editor.resetautofocus');
  }
  if (this.isCropEditMode()) {
    return i18n('editor.resetmask');
  }
}

return i18n('button.reset');
```